projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -78,6 +78,15 @@ export class ExplorerDashboardBuilder {
                 'scale-type': ScaleType.Band
               }
             }
+          : {}),
+        ...(isEmpty(request.groupBy) && isEmpty(request.interval)
+          ? {
```

Comment:
the conditions make it so only one (or none) of these will ever be added, but it's a bit complex to inline. Suggest moving out a function like `buildXAxis(request)`

Reply:
sure, let me update it